MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -219,9 +277,21 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification {
   [self didReceiveUserNotification:[notification.userInfo objectForKey:NSApplicationLaunchUserNotificationKey]];
 }
 
-- (void)userNotificationCenter:(NSUserNotificationCenter *)__unused center
+- (void)userNotificationCenter:(NSUserNotificationCenter *)center
        didActivateNotification:(NSUserNotification *)notification {
   [self didReceiveUserNotification:notification];
+  if ([self.originalUserNotificationCenterDelegate
+          respondsToSelector:@selector(userNotificationCenter:didActivateNotification:)]) {
+    [self.originalUserNotificationCenterDelegate userNotificationCenter:center didActivateNotification:notification];
+  }
+}
+
+- (void)forwardInvocation:(NSInvocation *)anInvocation {
+  if ([self.originalUserNotificationCenterDelegate respondsToSelector:[anInvocation selector]]) {
```

Comment:
Shouldn't we make sure the selector is part of the  NSUserNotificationDelegate protocol in addition to the responsiveness? Just to be sure we don't forward anything to the delegate.

Reply:
Good point.